docs/index.md

```diff
@@ -999,6 +999,18 @@ Options:
 
 The DigitalOcean driver will use `ubuntu-14-04-x64` as the default image.
 
+#### Generic
+Create machines using an existing VM/Host with SSH
```

Comment:
Missing a period here.  I'd also consider adding a little bit of explanation, for instance : "This is useful if you are using a provider Machine does not support directly or in the case that you want to use another tool to create the machines but import them into Docker Machine for usage".